sdk/vision/azure-ai-vision-imageanalysis/samples/run_all_samples.cmd

```diff
@@ -0,0 +1,16 @@
+REM
```

Comment:
was this for testing samples? Can we remove it?

Reply:
Running all the samples is something I do frequency before committing a change to a PR, in addition to running tests, so I use this script often. I would like to check it in so it is saved and may be used by others in the future. I also added a .ps1 version of this simple script so it can be run on all platforms.